workflow/cron/controller.go

```diff
@@ -266,29 +198,39 @@ func (cc *Controller) addCronWorkflowInformerHandler() {
 	})
 }
 
-func (cc *Controller) addWorkflowInformerHandler() {
-	cc.wfInformer.AddEventHandler(
-		cache.ResourceEventHandlerFuncs{
-			AddFunc: func(obj interface{}) {
-				key, err := cache.MetaNamespaceKeyFunc(obj)
-				if err == nil {
-					cc.wfQueue.Add(key)
-				}
-			},
-			UpdateFunc: func(old, new interface{}) {
-				key, err := cache.MetaNamespaceKeyFunc(new)
-				if err == nil {
-					cc.wfQueue.Add(key)
-				}
-			},
-			DeleteFunc: func(obj interface{}) {
-				key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
-				if err == nil {
-					cc.wfQueue.Add(key)
-				}
-			},
-		},
-	)
+func (cc *Controller) syncAll() {
```

Comment:
This is how K8s accomplishes this: https://github.com/kubernetes/kubernetes/blob/5beb14f0453c21b29f9252b1d04a4abb146d6028/pkg/controller/cronjob/cronjob_controller.go#L104